src/main/cms/cms_menu_vtx_msp.c

```diff
@@ -0,0 +1,287 @@
+{
+    UNUSED(pDisp);
+    UNUSED(self);
+
+    if (mspCmsPower == 0) {
+        // Bounce back
+        mspCmsPower = 1;
+    }
+
+    return NULL;
+}
+
+#define MSP_PIT_STATUS_NA (0)
+#define MSP_PIT_STATUS_OFF (1)
+#define MSP_PIT_STATUS_ON (2)
+
+static const char * const mspCmsPitModeNames[] = {
+    "---", "OFF", "ON "
+};
+
+static OSD_TAB_t mspCmsEntPitMode = { &mspCmsPitMode, 2, mspCmsPitModeNames };
+
+static const void *mspCmsSetPitMode(displayPort_t *pDisp, const void *self)
+{
+    UNUSED(pDisp);
+    UNUSED(self);
+
+    if (mspCmsPitMode == MSP_PIT_STATUS_NA) {
+        // Bouce back
+        mspCmsPitMode = MSP_PIT_STATUS_OFF;
+    } else {
+        vtxCommonSetPitMode(vtxCommonDevice(), (mspCmsPitMode == MSP_PIT_STATUS_OFF) ? 0 : 1);
+    }
+
+    return NULL;
+}
+
+static const void *mspCmsCommence(displayPort_t *pDisp, const void *self)
+{
+    UNUSED(pDisp);
+    UNUSED(self);
+
+    vtxDevice_t *device = vtxCommonDevice();
+    vtxCommonSetBandAndChannel(device, mspCmsBand, mspCmsChan);
+    vtxCommonSetPowerByIndex(device, mspCmsPower);
+
+    // update'vtx_' settings
+    vtxSettingsConfigMutable()->band = mspCmsBand;
+    vtxSettingsConfigMutable()->channel = mspCmsChan;
+    vtxSettingsConfigMutable()->power = mspCmsPower;
+    vtxSettingsConfigMutable()->freq = vtxCommonLookupFrequency(vtxCommonDevice(), mspCmsBand, mspCmsChan);
+
+    saveConfigAndNotify();
+
+    return MENU_CHAIN_BACK;
+}
+
+static bool mspCmsInitSettings(void)
+{
+    vtxDevice_t *device = vtxCommonDevice();
+    unsigned vtxStatus;
+
+    if (!device) {
+        return false;
+    }
+
+    vtxCommonGetBandAndChannel(device, &mspCmsBand, &mspCmsChan);
+
+    mspCmsUpdateFreqRef();
+    if (vtxCommonGetStatus(device, &vtxStatus)) {
+        mspCmsPitMode = (vtxStatus & VTX_STATUS_PIT_MODE) ? MSP_PIT_STATUS_ON : MSP_PIT_STATUS_OFF;
+    } else {
+        mspCmsPitMode = MSP_PIT_STATUS_NA;
+    }
+
+    if (!vtxCommonGetPowerIndex(vtxCommonDevice(), &mspCmsPower)) {
+        mspCmsPower = 1;
+    }
+
+    mspCmsEntBand.val = &mspCmsBand;
+    mspCmsEntBand.max = vtxTableBandCount;
+    mspCmsEntBand.names = vtxTableBandNames;
+
+    mspCmsEntChan.val = &mspCmsChan;
+    mspCmsEntChan.max = vtxTableChannelCount;
+    mspCmsEntChan.names = vtxTableChannelNames;
+
+    mspCmsEntPower.val = &mspCmsPower;
+    mspCmsEntPower.max = vtxTablePowerLevels;
+    mspCmsEntPower.names = vtxTablePowerLabels;
+
+    return true;
+}
+
+static const void *mspCmsOnEnter(displayPort_t *pDisp)
+{
+    UNUSED(pDisp);
+
+    if (!mspCmsInitSettings()) {
+        return MENU_CHAIN_BACK;
+    }
+
+    return NULL;
+}
+
+static const OSD_Entry mspCmsMenuCommenceEntries[] = {
+    { "CONFIRM", OME_Label,   NULL,          NULL },
+    { "YES",     OME_Funcall, mspCmsCommence, NULL },
+    { "NO",    OME_Back, NULL, NULL },
+    { NULL,      OME_END, NULL, NULL }
+};
+
+static CMS_Menu mspCmsMenuCommence = {
+#ifdef CMS_MENU_DEBUG
+    .GUARD_text = "XVTXMSC",
```

Comment:
VTXMSP ?

Reply:
I used convention found in other menus - X as start when in debug mode, C for Commence. I can change it to `MSPVTXC`.
https://github.com/betaflight/betaflight/blob/b745eaf28419cf8a86127ff9683fc121df43f69e/src/main/cms/cms_types.h#L117

Reply:
renamed